public/app/plugins/datasource/opentsdb/datasource.ts

```diff
@@ -40,6 +41,7 @@ export default class OpenTsDatasource extends DataSourceApi<OpenTsdbQuery, OpenT
     this.tsdbVersion = instanceSettings.jsonData.tsdbVersion || 1;
     this.tsdbResolution = instanceSettings.jsonData.tsdbResolution || 1;
     this.lookupLimit = instanceSettings.jsonData.lookupLimit || 1000;
+    this.interval = instanceSettings.jsonData.interval || '30s';
```

Comment:
This default would change all the queries because after this change everybody will default to 30s which can override other intervals.